src/traits.d

```diff
@@ -522,6 +546,15 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
     {
         return isFuncX(f => !f.needThis() && !f.isNested());
     }
+    if (e.ident == Id.isModule)
+    {
+        //TODO: can we handle the case when isPkgMod == PKGunknown?
```

Comment:
Maybe you have to handle `isPkgMod == PKGunknown` case for consistency.

Test case:

```d
module a;     // ./a.package.d
```
```d
module a.b;   // ./a/b.d
```
```d
module test;  // ./test.d
version(X) import a;
version(Y) import a.b;
pragma(msg, __traits(isModule, a));
// --> Prints true with -version=X, but prints false with-version=Y
```